CODEOWNERS

```diff
@@ -7,7 +7,7 @@ go.mod @fleetdm/go
 /cmd/ @fleetdm/go
 
 # Compliance
-/ee/cis/ @sharon-fdm @artemist-work @lucasmrod @marcosd4h
+/ee/cis/ @sharon-fdm @artemist-work @lucasmrod @marcosd4h @rachelElysia
```

Comment:
@sharon-fdm @lukeheath Do all 5 people need to receive notifications when a PR changes these files?  If not, then you can use 1 reviewer in CODEOWNERS and add multiple folks with auto-approve access:
https://fleetdm.com/handbook/company/why-this-way#reviewers

<img width="805" alt="image" src="https://user-images.githubusercontent.com/618009/226520492-bc027b35-1480-4118-bff3-0c0f0109fac4.png">

Reply:
@mikermcneil Specifically for the CIS work it's preferable.
The main reason is that CIS work creates a big number of PRs and we want several owners to review them while balancing their other work. 

Of course if you see a problem we can manually add people.

Reply:
Up to you